src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -274,6 +275,7 @@ private void readHeader() throws IOException {
     }
 
     //Visible for testing
+    @Deprecated
```

Comment:
Done, but because it is package specific I wasn't sure if it will be better to remove...